### PR TITLE
Initial commit from Create Next App

### PR DESCRIPTION
**Pull request by Rammfon**
hello world

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "react": "^18",
```
Comment (MrDoda) [inline package.json:12]: it's better to use exact versions. Rather than ^

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "react-dom": "^18",
```
Comment (MrDoda) [inline package.json:13]: Please add Prettier with some basic rules.

File: public/next.svg
Comment (MrDoda) [inline public/next.svg:1]: Remove unnacessary icons

**Review by MrDoda — CHANGES_REQUESTED** [review]

File: public/next.svg
Comment (Rammfon) [reply public/next.svg:1]: done

File: package.json (hunk 1 above)
Comment (Rammfon) [reply package.json:12]: done

File: package.json (hunk 2 above)
Comment (Rammfon) [reply package.json:13]: done

**Pull request merged**